src/modules/esl-popup/README.md

```diff
@@ -9,24 +9,94 @@ Authors: *Dmytro Shovchko*, *Alexey Stsefanovich (ala'n)*.
 <a name="intro"></a>
 
 **ESLPopup** is a custom element used as a wrapper for content that can be displayed as a pop-up GUI element.
-`ESLPopup` is based on `ESLToggleable` and works in pair with `ESLTrigger` that allows triggering ESLPopup instances state changes.
+`ESLPopup` is based on `ESLToggleable` and works in pairs with `ESLTrigger` which allows triggering ESLPopup instances state changes.
 
 ESLPopup can only function correctly if it's placed directly inside the `document.body`. 
 Therefore, when it connects to the DOM, it checks if it's in the body and, if it's not, 
 it moves itself to the `document.body` and leaves an `ESLPopupPlaceholder` element in its place. 
 
-### ESLPopup Attributes | Properties:
+### ESLPopup
 
-- `position` (string) - popup position relative to the trigger (currently supported: 'top', 'bottom', 'left', 'right' ) ('top' by default)
+What is a popup window? It is a rectangular area with content that is displayed above the main content of the page right next to the element that activated the popup. In other words, it's just a rectangle whose position relative to the trigger element is calculated according to some rules. To use a popup element with ease, you need to know these rules, so here is a brief description of them.
+
+#### Popups arrow
+
+A popup is an element that is displayed quite often along with an arrow that is focused on the trigger element that activated this popup. An arrow is a regular span element inside a popup that has a special class (see `ESLPopup Attributes` section for how to define a custom arrow class) on it. The `ESLPopup` element always checks for the presence of an arrow inside and creates it in case of its absence. So if you need a popup without an arrow, add the `disable-arrow` class to `ESLPopup` element.
```

Comment:
I disagree. The meaning of the sentence is wrong. Perhaps it would be better to rewrite it like this, do you agree?
```suggestion
A popup is an element that is displayed quite often, along with an arrow that focuses on the element that triggered the popup. An arrow is a regular span element inside a popup that has a special class (see `ESLPopup Attributes` section for how to define a custom arrow class) on it. The `ESLPopup` element always checks for the presence of an arrow inside and creates it in case of its absence. So if you need a popup without an arrow, add the `disable-arrow` class to `ESLPopup` element.
```